topics/dotnet/index.md

```diff
@@ -5,9 +5,9 @@ display_name: ".NET"
 github_url: https://github.com/dotnet
 logo: dotnet.png
 released: February 13, 2002
-short_description: ".NET is a software framework developed by Microsoft."
+short_description: ".NET is a free, cross-platform, open source developer platform."
```

Comment:
Need disambiguation. Only .NET Core is cross plat and open source. .NET Framework is neither.

Reply:
There is usually one ".NET" in the [introduction](https://dotnet.microsoft.com). .NET Core, .NET Framework, Xamarin and Mono are implementations of .NET.

Reply:
What are your thoughts @leecow?